src/components/Carousel.js

```diff
@@ -402,6 +430,16 @@ class Carousel extends Component {
             return currentPosition;
         }
```

Comment:
Does infinite loop work with centerMode?

Reply:
yes, it does now